src/scripts/__fixtures__/data-providers.ts

```diff
@@ -17,6 +17,15 @@ export const user: User = {
   deactivated: false,
 };
 
+export const deactivatedUser: User = {
```

Comment:
> Better to use the spread operator to re-use bits of base user.

How does the spread operator work? Which module is it from?

Reply:
Changes will be implemented. Didn't know the spread operator was the '...' operator